### ISSUE TITLE
Suggestion: Add an index page

### ISSUE THREAD

**Issue by gartz** [jwilder/nginx-proxy#725]
For development environments it would be cool to have by default a index page, that list all the virtual hosts available.

Going a bit advanced, we can add an env var to define the description of the host, like: `VIRTUAL_DESCRIPTION`.

**Comment by thomasleveil** [comment]
IMHO, that's too much trouble for low value. Bear in mind that such a feature would have to be maintained and that it won't work for all cases such as wildcard domains and domains matched by regexp.

**Comment by gartz** [comment]
True, maybe a link to a project that provides it and a note in the readme would be helpful and enough :)

**Comment by murbanowicz** [comment]
How can I create 301 to my company website when the VPS IP is called instead of giving 301 ?
